### PR TITLE
[CommonServerPython] CommandResults tagging support for the war room

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added tagging feature to CommandResults

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/ReleaseNotes/1_31_46.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Scripts
+##### CommonServerPython
+- added tagging capability when returning to war room
```
Comment (ilaner) [inline Packs/Base/ReleaseNotes/1_31_46.md:3]:
```suggestion
- Added support for `Tags` in `CommandResults` class in `CommonServerPython`.
```

File: Packs/Base/ReleaseNotes/1_31_34.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
 ##### CommonServerPython
-- Added a regular expression for domains.
+- Added a regular expression for domains.
```
Comment (ilaner) [inline Packs/Base/ReleaseNotes/1_31_34.md:4]: please revert

**Comment by ilaner** [comment]
Very good job!
Please resolve conflicts and see comment.

**Comment by ennozdd** [comment]
@ilaner done

**Review by ilaner — APPROVED** [review]

**Pull request merged**